config/proto/probe.proto

```diff
@@ -18,36 +31,38 @@ option go_package = "github.com/googleinterns/step224-2020/config/proto";
 // All Hermes probes should extend this probe config.
 message HermesProbeDef {
   string probe_name= 1;
-  TargetsDef targets = 2;
+  Targets targets = 2;
 
   // TargetSystem is expected to be a storage system.
   // It will stay constant until support for new storage systems is added.
   enum TargetSystem {
+    TARGET_SYSTEM_UNSPECIFIED = 0;
     GCS = 1;
     CEPH = 2;
   }
 
   // Probe types currently supported
   enum ProbeType {
+    PROBE_TYPE_UNSPECIFIED = 0;  // For unspecified probe types
     GET = 1;  // GET probe is for getting an object (file) from a storage system.
-    PUT = 2;  // PUT probe is for putting an object (file) into a storage system.  
+    PUT = 2;  // PUT probe is for putting an object (file) into a storage system.
     DELETE = 3;  // DELETE probe is for deleting an object (file) from a storage system.
   }
 
   TargetSystem target_system = 3;
   ProbeType probe_type = 4;
 
-  int32 interval_sec = 5 [default = 3600];  // Probing interval in seconds
+  int32 interval_sec = 5;  // Probing interval in seconds, default = 3600
 
```

Comment:
Why not an enum if we have a list of valid values?

Reply:
I am addressing comments on the .proto files in a different PR, as this branch is meant to only include the Cloudprober integration files. These .proto files are the same as those in the master branch.

Please see the new PR for the branch: evan/protoChanges.

--

I have changed the `latency_unit` into a enum.